Tests/Functional/Sync/SyncTest.php

```diff
@@ -158,8 +176,10 @@ private function getExpectedDocuments()
         $expectedDocument->addImageUrl('no_selection');
         $expectedDocument->addSmallImageUrl('no_selection');
 
-        $expectedDocuments['ONGRMagentoConnectorBundle:ProductDocument']['documents'][] = $expectedDocument;
+        $id = $expectedDocument->getId();
+        $expectedDocuments['ONGRMagentoConnectorBundle:ProductDocument']['documents'][$id] = $expectedDocument;
 
+        // Product With Category With Url.
```

Comment:
Comment is kinda the same as previous and i don't see that document has any setCategory or setUrl